engine/apps/mattermost/tests/test_representative.py

```diff
@@ -0,0 +1,100 @@
+import pytest
+from django.utils import timezone
+
+from apps.alerts.models import AlertGroupLogRecord, AlertReceiveChannel
+from apps.mattermost.alert_group_representative import AlertGroupMattermostRepresentative
+
+
+@pytest.mark.django_db
+def test_get_handler(
+    make_organization,
+    make_alert_receive_channel,
+    make_mattermost_channel,
+    make_alert_group,
+    make_alert,
+    make_alert_group_log_record,
+):
+    organization = make_organization()
+    alert_receive_channel = make_alert_receive_channel(
+        organization, integration=AlertReceiveChannel.INTEGRATION_GRAFANA
+    )
+    make_mattermost_channel(organization=organization, is_default_channel=True)
+    ack_alert_group = make_alert_group(
+        alert_receive_channel=alert_receive_channel,
+        acknowledged_at=timezone.now() + timezone.timedelta(hours=1),
```

Comment:
I see this a few times, any particular reason to set the ack_at in the future? (wondering if this could lead to unexpected things later)

Reply:
I saw this in couple of other places as well but i think the comment makes sense we can ack the alert at current timestamp